src/tutorials/lua_basics_lightbulb.md

```diff
@@ -203,79 +295,7 @@ Press **Play** and test it out!
-
-!["Trigger Location"](../img/LightBulb/image20.png){: .image-frame .center loading="lazy" }
-
-!!! hint "If you can't see a the trigger, press ++V++ to make it visible!"
-
-### Make the Trigger Interactable
-
-!["Interactable"](../img/LightBulb/image12.png "Interactable Box"){: .image-frame .center loading="lazy" }
-
-1. Look at the **Properties** window with the trigger selected. Under the **Gameplay** section there is a property called **Interactable**. Check the box to make the trigger interactable.
-2. Find the **Interaction Label** property, and change it to something like "Turn on Light"
-3. Press **Play** to start a preview, and walk up to the lightswitch. You should see a message
-
-### Script the Interaction
-
-Now that we know what a trigger is and where it is set up, we can get back to our script. We need to tell the script what our trigger is and what should happen when the player interacts with it.
-
-Add this line of code to the second line of your script,  after the `switch` variable:
-
-```lua
-local switchTrigger = switch:FindChildByName("Trigger")
-```
-
-- `switchTrigger` is the name for our trigger variable.
-- `switch:FindChildByName("Trigger")` looks for an object that is a child of the switch, and has the name "Trigger" in the Hierarchy.
-
-Your script should look like this:
-
-```lua
-local switch = script.parent
-local switchTrigger = switch:FindChildByName("Trigger")
-
-local onRotation = Rotation.New(0, -60, -0)
-local offRotation = Rotation.New(-180, -60, 180)
-
-switch:RotateTo(onRotation, 2, true)
-```
-
-### Test the Trigger Reference
-
-Getting in the habit of testing references will help you find bugs in code before they become a problem.
-
-Add this code on the third line of your script, after the line that makes the `switchTrigger` variable:
-
-```lua
-print(switchTrigger)
-```
-
-Press **Play** to start a preview, and see what prints in the **Event Log** window. You should see `Trigger` followed by the unique ID for your trigger object.
-
-![Print Trigger Reference](../img/LightBulb/updates/PrintTriggerReference.png){: .image-frame .center loading="lazy" }
-
-If you see `nil` instead, the most common reason is that your trigger is not in the same folder as the script. Look in your Hierarchy to check the following:
-
-- The trigger is in the **Switch (networked)** folder.
-- The trigger and **LightToggleScript** are in the same folder.
-- The trigger's name in the Hierarchy is **Trigger**.
-
-![Script and Trigger in the Switch Folder](../img/LightBulb/updates/ScriptAndTriggerInSameFolder.png){: .image-frame .center loading="lazy" }
-
-Once you know that the `switchTrigger` variable is pointing to the trigger in your game, you can delete the `print` line.
+So far, you have made a script that moves the light switch, but this happens automatically and immediately when the game starts. Next, you will make **the players** able to flip the switch to turn on and off our light.
```

Comment:
I think, stick to "You" rather than "We" throughout, and fix the earlier sections that use "We". It's a little clearer in translation, and a single subject.